app/webpacker/components/Results/Rankings/RankingsTable.jsx

```diff
@@ -57,69 +57,80 @@ function ResultRow({
         country = continents.real.find((c) => c.id === country.continentId);
       }
 
-      previousValue = value;
-      previousRank = rank;
-
-      return (
-        <ResultRow
-          country={country}
-          key={result.id}
-          result={result}
-          competition={competition}
-          rank={rank}
-          tiedPrevious={tiedPrevious}
-          isAverage={isAverage}
-          show={show}
-        />
-      );
-    });
-  }, [competitionsById, isAverage, rows, show]);
+      acc.push({
+        result,
+        competition,
+        country,
+        rank,
+        tiedPrevious,
+        key: `${result.id}-${show}`,
+      });
 
+      return acc;
+    }, []);
+  }, [competitionsById, rows, show]);
   return (
-    <Table basic="very" compact="very" singleLine striped>
-      <Table.Header>
-        {show !== 'by region' ? <Table.HeaderCell textAlign="center">#</Table.HeaderCell>
-          : <Table.HeaderCell>{I18n.t('results.table_elements.region')}</Table.HeaderCell>}
-        <Table.HeaderCell>{I18n.t('results.table_elements.name')}</Table.HeaderCell>
-        <Table.HeaderCell>{I18n.t('results.table_elements.result')}</Table.HeaderCell>
-        {show !== 'by region' && <Table.HeaderCell textAlign="left">{I18n.t('results.table_elements.representing')}</Table.HeaderCell>}
-        <Table.HeaderCell>{I18n.t('results.table_elements.competition')}</Table.HeaderCell>
-        {isAverage && (
-          <>
-            <Table.HeaderCell>{I18n.t('results.table_elements.solves')}</Table.HeaderCell>
-            <Table.HeaderCell />
-            <Table.HeaderCell />
-            <Table.HeaderCell />
-            <Table.HeaderCell />
-          </>
-        )}
-      </Table.Header>
-      <Table.Body>
-        {r}
-      </Table.Body>
-    </Table>
+    <div style={{ overflowX: 'scroll' }}>
+      <Table basic="very" compact="very" singleLine striped unstackable>
+        <Table.Header>
+          {show !== 'by region' ? <Table.HeaderCell textAlign="center">#</Table.HeaderCell>
+            : <Table.HeaderCell>{I18n.t('results.table_elements.region')}</Table.HeaderCell>}
+          <Table.HeaderCell>{I18n.t('results.table_elements.name')}</Table.HeaderCell>
+          <Table.HeaderCell>{I18n.t('results.table_elements.result')}</Table.HeaderCell>
+          {show !== 'by region'
+            && <Table.HeaderCell textAlign="left">{I18n.t('results.table_elements.representing')}</Table.HeaderCell>}
+          <Table.HeaderCell>{I18n.t('results.table_elements.competition')}</Table.HeaderCell>
+          {isAverage && (
+            <>
+              <Table.HeaderCell>{I18n.t('results.table_elements.solves')}</Table.HeaderCell>
+              <Table.HeaderCell />
+              <Table.HeaderCell />
+              <Table.HeaderCell />
+              <Table.HeaderCell />
+            </>
+          )}
+        </Table.Header>
+        <Table.Body key={show}>
```

Comment:
```suggestion
        <Table.Body>
```

Reply:
well this is what actually fixes the stuck table row issue